numpy/core/tests/test_nditer.py

```diff
@@ -34,6 +34,7 @@ def iter_iterindices(i):
         i.iternext()
     return ret
 
+@np.testing.dec.skipif(not HAS_REFCOUNT, "python does not have sys.getrefcount")
```

Comment:
Should be `@dec.skipif(not HAS_REFCOUNT, "python does not have sys.getrefcount")`, `dec` is imported above. That also brings the line length under the 79 character limit...